tests/test.py

```diff
@@ -0,0 +1,43 @@
+# flake8: noqa
```

Comment:
I think it is bad decision to ignore all flake rules. You can specify rules that you want to skip.